### DISCUSSION TITLE
Deploying and debugging to iOS doesn't work

### DISCUSSION THREAD

**Discussion by olivegamestudio** [unoplatform/uno#9140] [Q&A]
### Current behavior

Build project and then select mobile project as the startup project. Pick a simulator device from the next drop down. Picking net6.0-ios as the target framework.

Try to debug the project - it shows "deployed" but doesn't launch.

### Expected behavior

Expected to start the app in the simulator and debug it.

**Comment by jeromelaban** [comment]
This is the kind of issue we cannot fix on our end, unfortunately. If the app does not start it's generally a Visual Studio issue.

Still, you're likely to get additional information in the output tool window, in the "Xamarin" or "iOS" outputs. Which version of visual studio are you using?

**Reply by olivegamestudio** [reply]
I'll take a look in the logs to see if there's something useful. I'm using Visual Studio 2022 - 17.3.0 Preview 2.0.
